eoscostarica/eos-rate
# Fix/proxy logic

### GH Add Proxy list and Proxy Profile pages

#344 
#281 

### Steps to test

1. Go to EOSRATE web app
2. click sidebar menu and choose Proxies option
3. see proxies list page
4. click the "View" button on the proxy card
5. click on the Proxy card and go to profile
6. see all proxy information and block producers related to it
7. click on the "Vote" button

### Checklist
